verify/terraform/skus/current/windows-11-23H2/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
```

Comment:
verify/terraform/skus/current/windows-11-23H2/main.tf

It may be redundant "current" path of directory.

Reply:
Can we remove it?

Reply:
I'd prefer the following:

* No need to put it under "current" (skus/windows-11-xxx) is enough.
  * Reduce needless renaming for making it easier to track changelog
* Moving non supported version to "stale" is good idea.

Reply:
At least, with these refactoring, putting under sku are "supported" apparently, no need to add "current" prefix.

Reply:
Every comment you gave makes sense well.
This sounds like the matter of regarding how the folder structure /should/is going to/ be.

In my plan, I will eventually move skus that are outdated as well to rewrite them in this way.
So I still believe the "current" subfolder is worth to be.

In order to get to the agreement, maybe it is better to share my plan and discuss it.